projects/storybook-angular-examples/src/app/file/file-example.component.html

```diff
@@ -0,0 +1,54 @@
+<div style="max-width: 22rem">
```

Comment:
Je ne vois pas d'example pour mettre la traduction en anglais

Reply:
Il n'y en a pas, pour changer la langue du texte affiché il faut passer une classe custom contenant les textes en anglais comme provider pour BaoFileIntl dans le app.module.ts. Cela fera en sorte que chaque instance de BaoFileInput dans l'app affichera le texte en anglais, donc je ne vois pas comment faire un exemple de la composante en français et une en anglais au sein de la même app (storybook-angular-examples). Il est vrai toutefois qu'il n'y a aucune documentation à propos de comment utiliser BaoFileIntl pour avoir la composante en anglais, je ne savais pas comment intégrer cela à Storybook...

Reply:
Sinon mieux => https://stackoverflow.com/questions/64779479/storybook-inject-a-mocked-provider-service-at-component-level mais je n'ai pas testé

Reply:
Merci pour les suggestions, je vais essayer la solution proposée sur StackOverflow d'abord, et si ça ne fonctionne pas j'irai avec l'ajout d'une story d'étapes à suivre pour configurer la composante en anglais.

Reply:
@OlivierAlbertini J'ai finalement ajouté une story avec des étapes, pour éviter aux utilisateurs de devoir aller dans le code pour savoir comment changer la langue d'affichage. 
![baoFileIntl](https://user-images.githubusercontent.com/33531625/208728725-2ad7575d-1041-4064-bcc1-87fe05a1a1d5.PNG)

Reply:
Dans l'étape 2, je mettrais "Use this English translation (import { BaoFileIntlEnglish } from '@.../angular-ui') or declare your own class that ..."